tests/by-util/test_date.rs

```diff
@@ -401,6 +401,21 @@ fn test_invalid_date_string() {
         .stderr_contains("invalid date");
 }
 
+#[test]
+fn test_date_set_check_valid_format() {
```

Comment:
The function name is a bit misleading because the tests are unrelated to `--set`.

Reply:
@RenjiSann ping?

Reply:
thanks, will take care of this in the week :+1: